tests/nosetests/pyanaconda_tests/module_timezone_test.py

```diff
@@ -120,3 +128,218 @@ def kickstart3_test(self):
         timezone Europe/Prague --ntpservers=ntp.cesnet.cz
         """
         self._test_kickstart(ks_in, ks_out)
+
+
+class TimezoneTasksTestCase(unittest.TestCase):
+    """Test the D-Bus Timezone (Timezone only) tasks."""
+
+    def setUp(self):
```

Comment:
I think that you can remove the `setUp` method, because you don't need any of the attributes.